.travis.yml

```diff
@@ -1,25 +1,33 @@
 language: scala
+cache:
+  directories:
+    - $HOME/.ivy2
```

Comment:
Cool, I was wondering how to get the most of the caching with the new container based travis infra for open source projects. I'll give this a shot in some other projects too.